src/sectionproperties/post/fibre.py

```diff
@@ -0,0 +1,198 @@
+"""Provides functionalities to export a section to a fibre section.
+
+It can be used in [suanPan](https://github.com/TLCFEM/suanPan) to perform further FEA.
+"""
+from __future__ import annotations
+
+from abc import abstractmethod
+
+from numpy import ndarray
+
+from sectionproperties.analysis import Section, solver
+from sectionproperties.analysis.fea import Tri6, shape_function
+from sectionproperties.pre import CompoundGeometry, Geometry
+
+
+class Cell:
+    """Holds the information of a fibre cell."""
```

Comment:
Would it be possible to document the below attributes? It looks a bit strange on the generated doc page - see [here](https://sectionproperties--295.org.readthedocs.build/en/295/gen/sectionproperties.post.fibre.Cell.html#sectionproperties.post.fibre.Cell).

Can be done with:

```python
"""Holds the information of a fibre cell.

Attributes:
    tag: askdjaslkd
    area: aslkdjaskldj
    etc...
"""
```